WORKSPACE

```diff
@@ -59,3 +59,18 @@ bazel_skylib_workspace()
 
 load("@rules_pkg//:deps.bzl", "rules_pkg_dependencies")
 rules_pkg_dependencies()
+
+# For `kt_jvm_library`
+rules_kotlin_version = "v1.5.0-beta-4"
```

Comment:
This belongs in here:
https://github.com/protocolbuffers/protobuf/blob/master/protobuf_deps.bzl

Reply:
Moved the http_archive to protobuf_deps.bzl but I think I have to keep the registering toolchain calls in WORKSPACE, so I left those there.